unicodetools/data/ucd/dev/LineBreak.txt

```diff
@@ -2930,6 +2930,8 @@ FFFD           ; AI # So         REPLACEMENT CHARACTER
 11362..11363   ; CM # Mc     [2] GRANTHA VOWEL SIGN VOCALIC L..GRANTHA VOWEL SIGN VOCALIC LL
 11366..1136C   ; CM # Mn     [7] COMBINING GRANTHA DIGIT ZERO..COMBINING GRANTHA DIGIT SIX
 11370..11374   ; CM # Mn     [5] COMBINING GRANTHA LETTER A..COMBINING GRANTHA LETTER PA
+113D2          ; CM # Mn         TULU-TIGALARI GEMINATION MARK
+113D3          ; BA # Lo         TULU-TIGALARI SIGN PLUTA
```

Comment:
Why lb=BA "Break After" for this length mark? It looks like other length marks are lb=CM.

Reply:
See discussion with Ken:
> > You gave the Tulu-Tigalari sign pluta lb=ID, but the Grantha sign pluta, mentioned in the proposal, is lb=BA; is there a reason for lb=ID here?
>
> I don't really know what the correct assignments are here. The 3 plutas we have currently are inconsistent:
> 
> 1135D          ; BA # Lo         GRANTHA SIGN PLUTA
> 113D3          ; ID # Lo         TULU-TIGALARI SIGN PLUTA
> 11A9D          ; AL # Lo         SOYOMBO MARK PLUTA
> And there is a similar inconsistency for avagrahas, which are defaulting to AL for most scripts, but:
> 
> 0D3D           ; AL # Lo         MALAYALAM SIGN AVAGRAHA (etc., many others)
> 1133D          ; BA # Lo         GRANTHA SIGN AVAGRAHA
> 113B7          ; ID # Lo         TULU-TIGALARI SIGN AVAGRAHA
> L2/22-080R explicitly suggests AL->BA for the Grantha pluta and the Grantha avagraha. So I presume that both should 
> be treated the same way for Tulu-Tigalari, as well, for consistency. I will update those two.

Reply:
Help me understand. The doc (L2/22-260) says “Pluta denotes ‘extra-normal’ length for a vowel”. The handwritten examples seem to mostly show this in the middle of a word. (The printed example has an arrow pointing to a totally different glyph.) Why should there be a Break After the vowel length mark?

@Ken-Whistler

Reply:
Note the context of L2/22-080R; this is one of those scripts that break lines within words. We avoid breaking within orthographic syllables (though see the discussion in that doc, that might even be OK for Tulu-Tigalari), so not between the vowel and its lengthener. But after is fine.

Reply:
> Note the context of L2/22-080R; this is one of those scripts that break lines within words.

Thanks, that was what I was missing.